nextflow.config

```diff
@@ -24,39 +157,55 @@ params {
   plaintext_email = false
   monochrome_logs = false
   help = false
-  igenomes_base = 's3://ngi-igenomes/igenomes/'
+//  igenomes_base = 's3://ngi-igenomes/igenomes/'                                                   // pipeline does not support igenomes
   tracedir = "${params.outdir}/pipeline_info"
-  igenomes_ignore = false
-  custom_config_version = 'master'
-  custom_config_base = "https://raw.githubusercontent.com/nf-core/configs/${params.custom_config_version}"
-  hostnames = false
-  config_profile_description = false
-  config_profile_contact = false
-  config_profile_url = false
-
+//  igenomes_ignore = false                                                                         // pipeline does not support igenomes
   // Defaults only, expecting to be overwritten
   max_memory = 128.GB
   max_cpus = 16
   max_time = 240.h
 
 }
 
-// Container slug. Stable releases should specify release tag!
-// Developmental code should specify :dev
+
+//--------
+// Work directory:
+//--------
+workDir = "$baseDir/work"
+
+
+//--------
+// Container:
+//--------
 process.container = 'nfcore/dualrnaseq:dev'
 
 // Load base.config by default for all pipelines
 includeConfig 'conf/base.config'
 
+
 // Load nf-core custom profiles from different Institutions
 try {
   includeConfig "${params.custom_config_base}/nfcore_custom.config"
 } catch (Exception e) {
   System.err.println("WARNING: Could not load nf-core/config profiles: ${params.custom_config_base}/nfcore_custom.config")
 }
 
+// Load nf-core/dualrnaseq custom profiles from different Institutions
+try {
+  includeConfig "${params.custom_config_base}/pipeline/dualrnaseq.config"
+} catch (Exception e) {
+  System.err.println("WARNING: Could not load nf-core/config/dualrnaseq profiles: ${params.custom_config_base}/pipeline/dualrnaseq.config")
+}
+
+// to read your own custom genome file named computing_platform.config and saved in conf folder. 
+//includeConfig 'conf/computing_platform.config'                                                             
+
+// to read your own custom genome file named genomes.config and saved in conf folder. 
+//includeConfig 'conf/genomes.config'                                                                        
```

Comment:
I think you can move the test host and pathogen ref here (meaning in the conf/genomes.config), so that people will have an example to start with.

Reply:
I was talking to @bozmik about this. We see your point, but we'd like to keep this separate from the test run.
We agree though that this file should be provided, and will create a 'dummy' version so users can easily populate their desired host and pathogen files (will also add in comments to assist).
Does that work for you?

Reply:
sounds good to me

Reply:
https://github.com/nf-core/dualrnaseq/commit/769153c53efc7202f54a03c817e0926ca6985b5e

I added a draft of the genomes.config file and replaced igenomes parameters with genomes.config params in the nextflow.config

Reply:
Tidied up the genomes.conf file, giving examples, usage and documentation.